src/containers/GroupAdmin/components/AddMemberModal.tsx

```diff
@@ -0,0 +1,115 @@
+import { TextField, ListItemText } from '@material-ui/core';
+import useSnackbar from 'api/hooks/Snackbar';
+import { Controller, useForm } from 'react-hook-form';
+import { useUsers } from 'api/hooks/User';
+import { useMemo, useState } from 'react';
+import { Autocomplete } from '@material-ui/lab';
+import { getUserClass, getUserStudyShort } from 'utils';
+import { useCreateMembership } from 'api/hooks/Membership';
+import Dialog from 'components/layout/Dialog';
+
+type AddMemberModalProps = {
+  modalIsOpen: boolean;
+  handleClose: () => void;
+  groupSlug: string;
+};
+
+const AddMemberModal = ({ modalIsOpen, handleClose, groupSlug }: AddMemberModalProps) => {
+  const { control, handleSubmit } = useForm();
+  const showSnackbar = useSnackbar();
+  const createMembership = useCreateMembership();
+
+  const [search, setSearch] = useState('');
+  const filters = useMemo(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const filters: Record<string, any> = {};
```

Comment:
```suggestion
    const filters: Record<string, unknown> = {};
```

?? tror det er dette som er riktig